lib/clone_kit/cloners/active_record_merging_ruleset_cloner.rb

```diff
@@ -42,59 +43,59 @@ def merge(records)
 
       attr_accessor :merge_fields
 
-      def find_and_merge_existing_records(ids)
+      def find_and_merge_existing_records(ids, mergeable: Set.new, skip: skip = Set.new)
```

Comment:
I don't think it makes sense to move these into the method signature since we never use dependency injection here.